.github/workflows/lockdown.yml

```diff
@@ -0,0 +1,23 @@
+name: RepoLockdown
+on:
+  issues:
+    types: opened
+  pull_request_target:
+    types: opened
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/repo-lockdown@v2
+       with:
+          issue-comment: >
+            This repository does not accept bug reports,
+            see the README for details.
+          pr-comment: >
+            This repository does not accept pull requests,
+            see the README for details.
```

Comment:
We need it to not close new PRs, because we will continue to make release-3.8 PRs.  Unless you can set it to only close PRs on the master branch?

We do not want it to close existing open issues and PRs, not to add comments on closed issues and PRs.  Make sure those options are off (I'm not sure what the defaults are but I saw mentions of them).

Reply:
Yes for now I removed auto handling of the PRs and put it back to the old state where new PRs can be submitted and they will stay. 

This plugin will not impact the old issues that have been sitting there. It will auto close only the new PRs from the point this PR gets merged into master. Turns out its not sufficient of the plugin config is in a branch, it needs to be in the master branch only to work. Not sure if there is some other config which controlling this but for now we need to merge this into the master branch.